installer/resources/pacbot_app/files/DB.sql

```diff
@@ -2240,6 +2244,8 @@ INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType
+INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType,assetGroup,alexaKeyword,ruleParams,ruleFrequency,ruleExecutable,ruleRestUrl,ruleType,ruleArn,status,userId,displayName,createdDate,modifiedDate,severity,category) VALUES ('Enable_API_Key_Application_Restrictions','Enable_API_Key_Application_Restrictions','Enable_API_Key_Application_Restrictions','Enable API Key Application Restrictions change to Configure Application Restrictions for Google Cloud API Key','apikeys','gcp','Enable_API_Key_Application_Restrictions','{"params":[{"encrypt":false,"value":"Check-for-API-Key-Application-Restrictions","key":"ruleKey"},{"encrypt":false,"value":"medium","key":"severity"},{"encrypt":false,"value":"security","key":"ruleCategory"},{"encrypt":false,"value":"","key":"ruleOwner"}],"environmentVariables":[],"ruleId":"Enable_API_Key_Application_Restrictions","autofix":false,"alexaKeyword":"Enable_API_Key_Application_Restrictions","ruleRestUrl":"","targetType":"apikeys","pac_ds":"gcp","policyId":"Enforce_Separation_Of_Duties_For_KMS_Roles","assetGroup":"gcp","ruleUUID":"Enable_API_Key_Application_Restrictions","ruleType":"ManageRule"}','0 0/6 * * ? *',NULL,NULL,'ManageRule','arn:aws:events:us-east-1:***REMOVED***:rule/Enable_API_Key_Application_Restrictions','ENABLED','ASGC','Enable API Key Application Restrictions change to Configure Application Restrictions for Google Cloud API Key','2022-11-29','2022-11-29','medium','security');
```

Comment:
DisplayName value is exceeding the column limit of 100 char.
Please update the DDL for cf_RuleInstance to increase the displyName limit